pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -1364,55 +1114,20 @@ func TestAssignFlavors(t *testing.T) {
 				Usage: resources.FlavorResourceQuantities{},
 			},
 		},
-		"flavor not found": {
```

Comment:
Similar question as above - wondering if there could be race conditions before such a CQ is marked inactive so that it is worth covering with a test